docs/source/framework/pytorch_integration/layers_database.rst

```diff
@@ -0,0 +1,364 @@
+ML Layers database
```

Comment:
Ideally, this file should be auto-generated from headers with actual TC definitions, to avoid duplication and decoherence issues.

Reply:
yep, exactly and that requires some thinking, discussion and work